main.py

```diff
@@ -1,7 +1,49 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    # Create genres
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+
+    # Create actors
+    Actor.objects.create(first_name="George", last_name="Klooney")
```

Comment:
Typographical Error: The actor's last name 'Klooney' should be corrected to 'Clooney' initially to prevent the need for an update later.